llvm/lib/Transforms/Scalar/LoopStrengthReduce.cpp

```diff
@@ -1318,6 +1318,11 @@ class LSRUse {
   /// the loop, in which case some special-case heuristics may be used.
   bool AllFixupsOutsideLoop = true;
 
+  /// This records whether all of the fixups using this LSRUse are unconditional
+  /// within the loop, meaning they will be executed in every iteration of the
+  /// loop.
```

Comment:
I'd like some more precision here, something along these lines:
```suggestion
  /// within the loop, meaning they will be executed on every path to the loop latch.
  /// This includes fixups before early exits.
```